docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,229 @@
+            displayName: "Start Server Experiment",
+            path: "/server-experimentation/start",
+          },
+        ],
+      },
+    },
+    viewExperimentRun: {},
+  },
+  "@clutch-sh/server-experimentation": {
+    startExperiment: {
+      componentProps: {
+        upstreamClusterTypeSelectionEnabled: true,
+      },
+      hideNav: true,
+    },
+  },
+```
+
+#### Backend Server
+
+Below configuration will spin up all the required modules and services to store the data coming from frontend into Postgres database.
+
+```yaml title="backend/clutch-config.yaml"
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.api
+  - name: clutch.module.chaos.lyftexperimentation
+  - name: clutch.module.chaos.serverexperimentation
+services:
+  ...
+  - name: clutch.service.db.postgres
+      typed_config:
+        "@type": types.google.com/clutch.config.service.db.postgres.v1.Config
+        connection:
+          host: ${RDS_HOST}
+          port: ${RDS_PORT}
+          user: ${RDS_USER}
+          ssl_mode: REQUIRE
+          dbname: ${RDS_NAME}
+          password: ${CREDENTIALS_CLUTCH_TO_RDS_PASSWORD}
+  - name: clutch.service.chaos.experimentation.store
+```
+
+#### xDS Management Server 
+
+Below is the configuration for spinning up xDS server. For details about the fields, take a look at the [xds config proto](https://github.com/lyft/clutch/blob/main/api/config/module/chaos/experimentation/xds/v1/xds.proto).
+
+```yaml title="backend/clutch-xds-config.yaml"
+...
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.xds
+    typed_config:
+      "@type": types.google.com/clutch.config.module.chaos.experimentation.xds.v1.Config
+      rtds_layer_name: <RTDS_LAYER_NAME>                     // "rtds_layer"
+      cache_refresh_interval: <CACHE_REFRESH_INTERNAL>       // "5s"
+      ingress_fault_runtime_prefix: <INGRESS_FAULT_PREFIX>   // "fault.http"
+      egress_fault_runtime_prefix: <EGRESS_FAULT_PREFIX>     // "fault.http.egress"
+      resource_ttl: <RESOURCE_TTL>                           // "20s"
+      heartbeat_interval: <HEARTBEAT_INTERVAL>               // "5s"
+      ecds_allow_list: <LIST_OF_ECDS_ENALBED_CLUSTERS>       // ["foo", "bar"]
+services:
+  - name: clutch.service.db.postgres
+    typed_config:
+      "@type": types.google.com/clutch.config.service.db.postgres.v1.Config
+      connection:
+        host: ${RDS_HOST}
+        port: ${RDS_PORT}
+        user: ${RDS_USER}
+        ssl_mode: REQUIRE
+        dbname: ${RDS_NAME}
+        password: ${CREDENTIALS_CLUTCH_TO_RDS_PASSWORD}
+  - name: clutch.service.chaos.experimentation.store
+```
+
+Keep in mind that both backend config and xDS config needs to connect to the same Postgres database.
+
+### Example Envoy config
+
+When Envoy in the mesh boots up, it creates a bi-directional gRPC stream with the management server. Below is the sample Envoy configs for RTDS and ECDS which will initiate the connection to the xDS server 
+
+#### RTDS 
```

Comment:
maybe (just maybe) we could add copy-paste default fault-filter configuration and link to envoy docs for it so that they can learn about how default runtime keys look like etc.

Reply:
Added default fault filter in envoy config for RTDS and a link to Envoy Proxy fault injection doc